proposals/NNNN-Processing-of-unknown-enum-values-by-SDL-and-Mobile-libraries.md

```diff
@@ -197,17 +197,17 @@ If a parameter includes an unknown enum value, SDL Core has to cut off such valu
        AppHMIType" : ["DEFAULT", "MEDIA"]
        ```
 
-  ***Note**: In case no HMI types are left after filtering, Mobile library should throw Exception, otherwise RegisterAppInterface RPC will be sent with remaining HMI types.*
+  ***Note**: In case no HMI types are left after filtering, Mobile Library should throw an exception, otherwise RegisterAppInterface RPC will be sent with remaining HMI types.*
 
-  ***Note**: Now system flow is the following: if `RegisterAppInterface` fails with an error then Mobile library should pass the error status to the Mobile Library adding failure reason*
+  ***Note**: Now system flow is the following: if `RegisterAppInterface` fails with an error then Mobile library should pass the error status to the Mobile Library adding failure reason.*
 
-  ***Note**: The protocol version control logic partly implemented in Android and not implemented in iOS though it has its flaws and should be updated following the logic above.*
+  ***Note**: The protocol version control logic is partly implemented in Android and not implemented in iOS. However, what is implemented has its flaws and should be updated following the logic above.*
 
 
 ## Potential downsides
-  There might be certain corner cases with filtering out RPCs appeared since version greater than current.
+  In the case, new RPC's would added, the new structures/enums should be implemented with additional implementation of filtering mechanism for particular objects.
```

Comment:
```suggestion
  In the case new RPCs would added, the new structures/enums should be implemented with additional implementation of filtering mechanism for particular objects.
```

Reply:
@theresalech Updated. Please take a look at the f24753617739884847fba10c9ff538da4c3e1b98